units_cpp/tests/parse_json_test.cc

```diff
@@ -0,0 +1,13 @@
+#include "../game_rules.h"
+#include "../json.h"
+#include "catch.hpp"
+#include <iostream>
+
+using json = nlohmann::json;
+
+TEST_CASE("Parse GameRules") {
```

Comment:
Move to game_rules_test.cc

Reply:
Actually I'll get rid of this entirely since I don't think this tests anything that reading the same string from a file and parsing it doesn't.